### DISCUSSION TITLE
Unable to upload attachment files. Err:[provisioning_quota_exceeded]

### DISCUSSION THREAD

**Discussion by Mohit-3430** [padloc/padloc#455] [Q&A] [answered]
There is feature in padloc where users can upload their files, so in my development environment (wsl 2) I am using files system **fs** as storage medium out of availbale oprions. But when I am trying to upload files via Web interface I am facing this issue:

```
@padloc/server: Err [Error]: You have reached the file storage limit for this account!
@padloc/server:     at C.createAttachment (/home/mo_linux/Projects/padloc/packages/core/src/server.ts:1513:23)
@padloc/server:     at C.process (/home/mo_linux/Projects/padloc/packages/core/src/server.ts:236:24)
@padloc/server:     at Server.handle (/home/mo_linux/Projects/padloc/packages/core/src/server.ts:1917:31)
@padloc/server:     at Server.<anonymous> (/home/mo_linux/Projects/padloc/packages/server/src/transport/http.ts:77:37) {
@padloc/server:   time: 2022-06-02T11:22:56.316Z,
@padloc/server:   code: 'provisioning_quota_exceeded',
@padloc/server:   report: false,
@padloc/server:   display: false,
@padloc/server:   originalError: undefined
@padloc/server: }
```

**Comment by BrunoBernardino** [comment, marked as answer]
@Mohit-3430 you can change the default quota values by changing these numbers: 
https://github.com/padloc/padloc/blob/17df06a14d4618049a4344a810547f5599cc08c3/packages/core/src/provisioning.ts#L35-L36

If you're not using an Org, you might need to update these instead: 
https://github.com/padloc/padloc/blob/17df06a14d4618049a4344a810547f5599cc08c3/packages/core/src/provisioning.ts#L23-L26

@MaKleSoft is there a way to increase/set these via config? I didn't find it.

Let me know if that works!

**Edit** switched out the code references -M

**Reply by MaKleSoft** [reply]
Nope, there is no way to change default quotas through the config at this point. You could create a provisioning database entry with `[default]` as a key, but that may be a bit cumbersome to do by hand (it will be easier once we get the admin portal ready). So for now changing the defaults in the code is probably the easiest way.

@BrunoBernardino actually, we may want to change the default storage quota to `-1` for both orgs and accounts (`-1` means unlimited). And while we're at it, do that for all other quota values as well, since I don't see a reason why these should be limited by default.

**Reply by Mohit-3430** [reply]
Ok now I understood you have limited the quotas, for subscription model to work. So for developing purpose will you always change that numbers in [this file](https://github.com/padloc/padloc/blob/17df06a14d4618049a4344a810547f5599cc08c3/packages/core/src/provisioning.ts#L35-L36) ..??
or there is another alternative..??

**Reply by Mohit-3430** [reply]
@BrunoBernardino I tried changing all the values as u have mentioned to `-1` suggested by @MaKleSoft , unfortunately I got the same error again.

**Reply by MaKleSoft** [reply]
@Mohit-3430 Can you try deleting and recreating the account you're testing with (or test with a new one)?

**Reply by Mohit-3430** [reply]
Yeah, It Worked @MaKleSoft

**Reply by Mohit-3430** [reply]
Hey I see there is stripe provisioning if we apply that, there will be no change in [this file](https://github.com/padloc/padloc/blob/17df06a14d4618049a4344a810547f5599cc08c3/packages/core/src/provisioning.ts#L35-L36)..??
because I don't think it is a good practice to change the values for development, if I do a PR here i need to change that before i push right so ..??

**Reply by MaKleSoft** [reply]
@Mohit-3430 Yep, you're right. That's why I'm saying we should change the default to no limits, so that it's easier to set up for development or a self-hosted, private instance. Just created an issue for that: https://github.com/padloc/padloc/issues/460

**Reply by Mohit-3430** [reply]
ok @MaKleSoft perfect thanks.
